src/common/user.c

```diff
@@ -24,6 +24,7 @@ static struct
 
     gid_t groupId;                                                  // Real group id of the calling process from getgid()
     const String *groupName;                                        // Group name if it exists
+    const String *userHome;                                         // User HOME directory
```

Comment:
Should this also be under a #ifdef HAVE_LIBSSH2 similar to the other code..?  Otherwise some part of the code might try to use it without being compiled with HAVE_LIBSSH2 and it'd be uninitialized...

Reply:
It should. I'll make it so.